app/src/controllers/userController.ts

```diff
@@ -0,0 +1,129 @@
+import mongoose from 'mongoose';
+import { UserSchema } from '../models/userModel';
+import { Request, Response } from 'express';
+
+const User = mongoose.model('User', UserSchema);
+
+/**
+ * CRUD-Controller for user models
+ */
+export class UserController {
+  /**
+   * Lists all available users
+   * @param  {Request} req - express request object
+   * @param  {Response} res — express response object
+   * @return json object with statusCode and list of users
+   */
+  public async index(req: Request, res: Response) {
```

Comment:
need to specify return type in all methods

Reply:
You mean declaring it like: create(req: Request, res: Response): void
I guess in this case void would be the most appropriate for all the methods?
or in the docblock like: @return {TYPE} json
Here I'm not sure what type that would be?